gobblin-core/src/main/java/org/apache/gobblin/converter/avro/JsonElementConversionFactory.java

```diff
@@ -432,16 +470,54 @@ protected void setElementConverter(JsonElementConverter elementConverter) {
     public JsonElementConverter getElementConverter() {
       return this.elementConverter;
     }
+
+    protected void processNestedItems(String fieldName, JsonObject schemaNode, WorkUnitState state,
+        JsonElement arrayItems)
+        throws UnsupportedDateTypeException {
+      if (arrayItems.isJsonPrimitive()) {
+        this.setElementConverter(
+            getConvertor(fieldName, "", arrayItems.getAsString(), getDataType(schemaNode), state, isNullable()));
```

Comment:
Can use the other constructor here, and in a couple other places where it's called with namespace as "".

Reply:
👍